buildbot-host/scripts/install-openvpn-build-deps-ubuntu.sh

```diff
@@ -81,6 +80,9 @@ uuid-dev
 # Only for some distros
 $APT_INSTALL systemd-dev || true
 
+# policykit-1 package was split to two packages in Debian unstable in late 2024
```

Comment:
That comment is incorrect. The package split is already in Debian stable. What happened now is that the transitional package depending on both of them was removed.

Reply:
Fixed by making the statement more generic: "policykit-1 is not available in more recent operating systems".